### PR TITLE
QMSI v1.1.0  ALPHA  drop

### PR DESCRIPTION
**Pull request by jeez**
This pull request brings QMSI v1.1.0 ALPHA code drop and updates the README file.

What is pending after this:
- Update the Known Issues section
- Link to new toolchain release and also to new ISSM when they go out
- Update flashing instructions for Quark SE Devboard

THIS IS AN ALPHA RELEASE!

### CODE REVIEW

**Comment by nashif** [comment]
you should remove all those whitespaces in the license headers   :)

**Comment by plmorale** [comment]
Actually this is a bit strange, the original license came without whitespaces, I'll track where they got inserted

File: README.rst [hunk 1]
```diff
@@ -214,15 +214,34 @@ Supported features
```
Comment (plmorale) [inline README.rst]: Can we squash all README changes together.
Also I think we should be inline with the release notes produced by TMEs
(I'll forward that to you)

**Comment by plmorale** [comment]
Can we squash all README changes together?
Also I think we should be inline with the release notes released with ISSM.

**Comment by jeez** [comment]
@nashif @plmorale I think the white spaces were inserted by the pack_bsp.py script . I will have a look.

@plmorale I can squash the README changes if you want. I was just making it easier for you to reviews the changes. :)  I used the release notes from ISSM and basically copied what we needed from there into our README.md . It's a very manual process. Actually, ideally we would have one being generated from the other or we could just have our README.md linking to some online documentation somewhere. I just don't think a .docx file is the best media for that, to be honest.

**Comment by nashif** [comment]
You have files from 1.0.1 that are not part of 1.1 anymore still in the tree, for example: qm_scss.c. this file was removed in 1.1

**Comment by jeez** [comment]
@nashif @plmorale  I just fixed it now. Can you please have another look?

The commits for the README file were squashed as requested.

**Comment by jeez** [comment]
For the sake of documentation, this Pull Request is currently pending on a newer toolchain release.
@plmorale should I just merge it ?

**Pull request merged**